app/src/main/res/layout/list_item_submission_left.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<me.saket.dank.widgets.swipe.SwipeableLayout android:id="@+id/submission_item_root"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:background="@color/window_background">
+
+    <me.saket.dank.widgets.swipe.SwipeActionIconView style="@style/DankSwipeableLayoutIcon" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:elevation="2dp"
+        android:foreground="?attr/selectableItemBackground"
+        android:orientation="vertical"
+        android:paddingTop="@dimen/spacing12">
+
+        <RelativeLayout
+            android:id="@+id/submission_item_content_container"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/submission_item_title"
+                android:layout_width="610dp"
+                android:layout_height="wrap_content"
+                android:layout_alignStart="@+id/submission_item_icon"
+                android:layout_alignTop="@+id/submission_item_icon"
+                android:layout_alignParentEnd="true"
+                android:layout_marginStart="67dp"
```

Comment:
This and other margins don't look ok... They seem a bit too specific. Please check values in original file (Check the XML tab and not the preview tab) and try to make yours as close to them as possible.

Reply:
Fixed.